src/main/resources/db/migration/V21__ReadCaseInteractionPermission.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO permission (id, role_id, permission_type)
```

Comment:
need to add READ_USER_INTERACTIONS to super-user also

Reply:
Agreed